src/main.cpp

```diff
@@ -710,7 +708,8 @@ int main(int argc, char *argv[])
 
   TimeThis tt("main() to doOneRun()");
 
-  SetThisThreadName("main");
+  env::prependToPath(QDir::toNativeSeparators(
```

Comment:
Looks like a bad merge, `SetThisThreadName()` is missing. It's also redundant, it's already done in `addDllsToPath()`.